cli/src/main.rs

```diff
@@ -4,13 +4,13 @@ use std::process::Command;
 
 use base64;
 use clap::{crate_name, crate_version, App, AppSettings, SubCommand};
-use log::error;
 use serde_json::json;
 
 use anycloud::deploy::{client_error, get_config, info, new, terminate, upgrade, ALAN_VERSION};
+use anycloud::error;
```

Comment:
you can instead do something like this at the top of the file in case we end up adding more macros:
```rs
#[macro_use]
extern crate anycloud;
```